TreeTracker/UI/Views/Home/HomeViewController.swift

```diff
@@ -113,4 +114,18 @@ extension HomeViewController: HomeViewModelViewDelegate {
     func homeViewModel(_ homeViewModel: HomeViewModel, didReceiveError error: Error) {
         present(alert: .error(error))
     }
+
+    func homeViewModel(_ homeViewModel: HomeViewModel, didFetchProfile data: Data) {
+        // use custom button to fit our image
+        let profileImage = UIImage(data: data)
+        let profileButton = UIButton(type: .custom)
+        profileButton.setImage(profileImage, for: .normal)
+        profileButton.layer.cornerRadius = 20
+        profileButton.layer.masksToBounds = true
+        let profileBarButton = UIBarButtonItem(customView: profileButton)
```

Comment:
Would `UIBarButtonItem(image:...)`suffice rather than creating UIButton?
e.g.
```
let button = UIBarButtonItem(
    image: Asset.Assets.person.image,
    style: .plain,
    target: nil,
    action: nil
)
navigationItem.leftBarButtonItem = button
```

I think also it may worth creating and adding the button once, maybe in viewDidLoad then just updating the button image here.